.github/workflows/pull_request.yml

```diff
@@ -23,29 +23,20 @@ jobs:
     runs-on: ${{ matrix.os }}
     name: Maven Build
     steps:
-      - name: Setup Maven And Java Version
-        uses: s4u/setup-maven-action@v1.2.1
+      - name: Support long paths
+        if: ${{ matrix.os == 'windows-latest' }}
+        uses: kiegroup/droolsjbpm-build-bootstrap/.ci/actions/long-paths@main
+      - name: Java and Maven Setup
+        uses: kiegroup/droolsjbpm-build-bootstrap/.ci/actions/maven@main
         with:
           java-version: ${{ matrix.java-version }}
           maven-version: ${{ matrix.maven-version }}
-      # See https://docs.github.com/en/actions/guides/building-and-testing-java-with-maven#caching-dependencies
-      - name: Cache Maven packages
-        uses: actions/cache@v2
+          key-prefix: ${{ runner.os }}-${{ matrix.java-version }}-maven${{ matrix.maven-version }}
```

Comment:
it should be `cache-key-prefix` on all PRs ;)

Reply:
@radtriste done! thanks